fxa-rust-client/src/http_client/browser_id/jwt_utils.rs

```diff
@@ -97,7 +97,7 @@ impl<'keypair> SignedJWTBuilder<'keypair> {
         };
         let obj = match payload.as_object_mut() {
             Some(obj) => obj,
-            None => bail!("Invalid JSON"),
+            None => panic!("The supplied payload was not an object"),
```

Comment:
Does panicking here make sense? (I guess fxa’s ffi is unwind safe so it might, but could rust consumers want to handle this?)

Reply:
I think it makes sense since it's a programming error (since we provide the JSON value), but I'm ok with reverting back to an exception.